examples/rust/custom_space_view/src/color_coordinates_space_view.rs

```diff
@@ -225,22 +228,21 @@ fn color_space_ui(
 
             let interact = ui.interact(
                 egui::Rect::from_center_size(center, egui::Vec2::splat(radius * 2.0)),
-                ui.id().with(("circle", i)),
+                ui.id().with(("circle", instance_key, ent_path.hash())),
```

Comment:
small things: keep hierarchy order, and no need for .hash() - `with` hashes its argument, that's what it does :)

```suggestion
                ui.id().with(("circle", &ent_path, instance_key)),
```

Reply:
nice!